.pre-commit-config.yaml

```diff
@@ -8,13 +8,6 @@ repos:
         args: ["--fix=lf"]
       - id: trailing-whitespace
 
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.7.1
-    hooks:
-      - id: prettier
-        types_or: [css, html, javascript, json, jsx, markdown]
-        exclude: ^frontend/public/assets
-
```

Comment:
I'm removing this precommit hook because it does the same thing as `npm run format`, and it conflicts with having the `prettier-plugin-tailwindcss` in `.prettierrc.json`.

To use plugins with precommit prettier, you're supposed to add this: 
```
additional_dependencies:
  - prettier
  - prettier-plugin-tailwindcss
```
to the precommit config, but this still errors if `"plugins": ["prettier-plugin-tailwindcss"]` is included in `prettierrc.json`.

I would prefer to have `"plugins": ["prettier-plugin-tailwindcss"]` in `prettierrc.json` than have this pre-commit hook.

Reply:
we still have a check in github actions for PRs right? that cathes the same things
just double-checking (cuz I'm in a rush oop i can check later)